CHANGELOG.md

```diff
@@ -73,6 +73,7 @@
 * [BUGFIX] Frontend, Query-scheduler: allow querier to notify about shutdown without providing any authentication. #4066
 * [BUGFIX] Querier: fixed race condition causing queries to fail right after querier startup with the "empty ring" error. #4068
 * [BUGFIX] Compactor: Increment `cortex_compactor_runs_failed_total` if compactor failed compact a single tenant. #4094
+* [BUGFIX] Fixed cache fetch error on Redis Cluster. #4056
```

Comment:
Please rebase `master` and move this CHANGELOG entry to the top. We've cut 1.9 release in the meanwhile. Thanks!

Reply:
@kamijin-fanta This commit is still valid. Could you address it, please?